aggregator/programs/build.ts

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env -S deno run --unstable --allow-run --allow-read --allow-write
+#!/usr/bin/env -S deno run --unstable --allow-run --allow-read --allow-write --allow-env
```

Comment:
I'm not sure how this worked before. Possibly due to deno upgrades. Anyway, `build.ts` accesses environment variables, so it needs this flag.